packages/dtslint/src/lint.ts

```diff
@@ -1,7 +1,7 @@
 import { TypeScriptVersion } from "@definitelytyped/typescript-versions";
 import { typeScriptPath } from "@definitelytyped/utils";
 import assert = require("assert");
-import { pathExists } from "fs-extra";
+import { pathExistsSync } from "fs-extra";
```

Comment:
Not for this PR, but modern Node's FS lib is rich enough that I suspect we could drop fs-extra.